static_frame/core/store_zip.py

```diff
@@ -173,22 +206,56 @@ def read_many(self,
             ) -> tp.Iterator[Frame]:
 
         config_map = StoreConfigMap.from_initializer(config)
+        single_config = config_map.is_empty()
+        frames_parameters = []
 
+        # Load in all byte & kwarg information needed to build the frames
         with zipfile.ZipFile(self._fp) as zf:
             for label in labels:
                 c = config_map[label]
-                label_encoded = config_map.default.label_encode(label)
 
+                label_encoded = config_map.default.label_encode(label)
                 src = BytesIO(zf.read(label_encoded + self._EXT_CONTAINED))
-                yield container_type.from_parquet(
-                        src,
+
+                src_and_name = src, label
+
+                if single_config:
+                    # No need to duplicate config info here
+                    frames_parameters.append(src_and_name)
```

Comment:
`c` in this context will be the default config, if no per-frame configs were defined, so we do not need this branch, I think.